doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -93,7 +93,18 @@ Matter
 * For devices that use Matter over Thread, the default cryptography backend is now Arm PSA Crypto API instead of Mbed TLS, which was used in earlier versions.
   You can still build all examples with deprecated Mbed TLS support by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA` Kconfig option to ``n``, but you must build the Thread libraries from sources.
   To :ref:`inherit Thread certification <ug_matter_device_certification_reqs_dependent>` from Nordic Semiconductor, you must use the PSA Crypto API backend.
-* Updated the page about :ref:`ug_matter_device_low_power_configuration` with the information about Intermittently Connected Devices (ICD) configuration.
+
+* Updated:
+
+  * The :ref:`ug_matter_device_low_power_configuration` page with the information about Intermittently Connected Devices (ICD) configuration.
+  * The Factory Data Guide and added a new section explaining several important Kconfig options.
+
+* Changed the factory data default usage:
+
+  * The SPAKE2+ verifier is now generated by default with each build, and it will change if any of the related Kconfig options are modified.
+    This resolves the :ref:`known issue <known_issues>` related to the SPAKE2+ verifier not regenerating (KRKNWK-18315).
+  * The Test Certification Declaration can now be generated independently and no longer necessitates the generation of the DAC and PAI certificates.
+  * The Rotating Device ID Unique ID can be utilized and generated only if the :kconfig:option:`CONFIG_CHIP_ROTATING_DEVICE_ID` Kconfig option is set to ``y``.
```

Comment:
Can you ellaborate more on this point. From description i don't understand what has changed, and RDID UID could be utilized and generated before. Maybe we can mention, that some KConfig is no longer needed now or sth.

Reply:
We could change this line to:
```suggestion
  * The Rotating Device ID Unique ID can now only be utilized and generated if the :kconfig:option:`CONFIG_CHIP_ROTATING_DEVICE_ID` Kconfig option is set to ``y``.
```
Adding "now" and moving "only" up a bit should make it clearer.